src/components/mols/EmptyCoupons/index.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+import { StaticImport } from "next/dist/shared/lib/get-img-props";
+import { LoadingComponent } from "@/components/atoms";
+interface IEmptyCoupons {
+  couponColor: StaticImport;
+  title: string;
+  label?: string;
+  href?: string;
+}
+
+const EmptyCoupons: React.FC<IEmptyCoupons> = ({
+  couponColor,
+  title,
+  label,
+  href,
+}) => {
+  const [stateLabel, setStateLabel] = useState<any>(label);
+
+  const onClick = () => {
+    setStateLabel(<LoadingComponent fullSize={false} bgColor="none" />);
+  };
```

Comment:
- Again this name "stateLabel". What does that meaaan? Plsss helpppp 
- Make it simples, use isLoading

Reply:
@femacedogomes this sugestion has ben solved??

Reply:
Replace all "setStateLabel" logic where u use in app; Make more simples.